tests/utils.ts

```diff
@@ -37,10 +38,44 @@ export const test = baseTest.extend({
   },
 })
 
-export function getYesterday(): string {
-  const yesterday = moment().subtract(1, 'days')
-  const formattedDate = yesterday.format('DD/MM/YYYY')
-  return formattedDate
+export function getPastDate(): Date {
+  return moment('2024-02-12 15:00:00').toDate()
+}
+
+export async function setBrowserTime(date: Date, page: Page | BrowserContext) {
+  const fakeNow = date.valueOf()
+
+  // Update the Date accordingly
+  await page.addInitScript(`{
```

Comment:
suggestion - you can pass a function instead of a string to enjoy linting and intellisense

Reply:
updated!